src/index.njk

```diff
@@ -2,4 +2,8 @@
 title: Index
 ---
 
+PNG+GIF
 {% photoFolder "src/assets/images" %}
+
+JPEG-only
+{% photoFolder "src/assets/images", [".jpg", ".jpeg"] %}
```

Comment:
But this random example only shows *.jpg and *.jpeg images.

Probably best just to list the 2-6 file types you want to support in the shortcode directly since trying to separate by extension might get weird.